### ISSUE TITLE
Keyboard focus does not move to the filter controls on the second attempt when the filter panel is expanded in Windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/kendo-react#3582]
### Description

**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7670)
Browsers: Edge (Version 145.0.3800.22), Chrome (Version 144.0.7559.110)
URL: https://www.telerik.com/kendo-react-ui/components/grid

**User Impact:**
Keyboard-only users may not be able to reach the filter controls consistently, as focus fails to move to the controls on the second attempt when the filter panel is expanded. This can prevent them from adjusting filters independently and disrupts smooth navigation.

### Plan

KendoReact Free

### Steps To Reproduce

1. Open the URL: https://www.telerik.com/kendo-react-ui/components/grid in edge browser in windows.
2. 'React Data Grid Overview' page will open.
3. Press tab key to reach to the 'More Option' button and activate it.
4. Press tab key to reach to the 'Filters' control and activate it.
5. Filters controls will appear.
6. Press tab key to navigate to the filter controls.
7. Verify whether Keyboard focus moves to the filter controls on the second attempt or not.

### Screenshots or video

**Refer Attachment:**

1. Keyboard focus does not move to the filter controls on the second attempt in Windows.png
2. Keyboard focus does not move to the filter controls on the second attempt in Windows.mp4

https://github.com/user-attachments/assets/8b13bfaf-cb37-4bc3-bf41-fc9cc6ef0647
<img width="975" height="607" alt="Image" src="https://github.com/user-attachments/assets/d5ea3f61-1cde-4f7d-ad2b-c37f646307d2" />

### Actual Behavior

Keyboard focus does not move to the filter controls on the second attempt when the filter panel is expanded in Windows.

### Expected Behavior

Keyboard focus should move to the filter controls on the second attempt when the filter panel is expanded in Windows.

### Browser

Chrome, Microsoft Edge

### Browser version

Edge (Version 145.0.3800.22), Chrome (Version 144.0.7559.110)

### OS type

Windows

### OS version

Windows 11 version 25H2 (OS Build 26220.7670)

**Comment by PekoPPT** [comment]
Hi, @msftedad. 
I just tested the reported behavior, and I can confirm that the scenario described above is a bug. I am marking the current issue as a bug. Thank you for your report!

**Comment by filipKovachev** [comment]
Hello, @msftedad,

After further inspection, it appears that this bug has already been logged here: https://github.com/telerik/kendo-react/issues/3409. Based on your report, I have increased its priority. I will close this issue as a duplicate, but you can continue to track the progress in the original thread.
